strongbox-parent/pom.xml

```diff
@@ -69,7 +69,7 @@
         <version.jetty>9.4.6.v20170531</version.jetty>
         <version.lucene>4.8.1</version.lucene>
         <version.logback>1.2.3</version.logback>
-        <version.maven>3.3.9</version.maven>
+        <version.maven>3.6.0</version.maven>
```

Comment:
If we're changing this, then we'll also need to update the [Requirements](https://github.com/strongbox/strongbox#requirements) section of the `README.md` accordingly.

Reply:
Let me know if this is a requirement so I can upgrade our ci docker images and jenkins jobs (where necessary).

Reply:
It should not be necessary. I'll roll it back

Reply:
Okay, many thanks! :)